CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[BREAKING CHANGE]** Migrate `DatePicker` to ReactDayPicker and remove React Dates
```

Comment:
It may be useful to develop a bit: 
- Which properties are removed
- Removing forced dependency to moment

Reply:
I added a bit more details on what changed.